cm_sdk/api/__init__.py

```diff
@@ -0,0 +1,4 @@
+__author__ = 'e0d'
+
+class APIException(Exception):
```

Comment:
Right, for *this* API, but users could be using other APIs. I'm just suggesting something like `CloudmanagerApiException`